### PR TITLE
Expose Folder and Label CRUD actions

### PR DESCRIPTION
**Pull request by zspencer**

### CODE REVIEW

File: lib/nylas/api.rb [hunk 1]
```diff
@@ -40,6 +40,16 @@ def accounts
       @accounts ||= Collection.new(model: Account, api: as(client.app_secret))
     end
 
+    # @return [Collection<Folder>] A queryable collection of Accounts
+    def folders
+      @accounts ||= Collection.new(model: Folder, api: self)
```
Comment (khamidou) [inline lib/nylas/api.rb]: Hmm, not sure I understand the `@accounts ||=` part.

File: spec/fixtures/opensource_accounts_endpoint.txt [hunk 2]
```diff
@@ -1,11 +0,0 @@
-[
```
Comment (khamidou) [inline spec/fixtures/opensource_accounts_endpoint.txt]: Did you mean to delete all these fixtures?

File: lib/nylas/api.rb (hunk 1 above)
Comment (zspencer) [reply lib/nylas/api.rb]: Copy-paste error. Good catch!

File: spec/fixtures/opensource_accounts_endpoint.txt (hunk 2 above)
Comment (zspencer) [reply spec/fixtures/opensource_accounts_endpoint.txt]: Yeah, we aren't using them anymore. Instead, we're putting the fixtures inline in the tests so it's more clear what the inputs/outputs are.

**Pull request merged**